tests/xontrib_chatgpt/test_chatgpt.py

```diff
@@ -122,22 +130,24 @@ def test_chat_raises_error_with_no_chat_model(xession, chat, monkeypatch_openai)
         chat.chat("test")
 
 
+
 def test_chat_catches_openai_errors(xession, chat, monkeypatch):
     xession.env["OPENAI_API_KEY"] = "test"
     xession.env["OPENAI_CHAT_MODEL"] = "gpt-3.5-turbo"
 
+
     def raise_it(*_, **__):
         raise RateLimitError("test")
 
-    monkeypatch.setattr(
-        "xontrib_chatgpt.chatgpt.openai.ChatCompletion.create", raise_it
-    )
+    monkeypatch.setattr("xontrib_chatgpt.chatgpt.openai.completions.create", raise_it)
     with pytest.raises(SystemExit):
         chat.chat("test")
 
 
+
 def test_chat_convo(xession, chat):
     assert chat.chat_convo == chat.base
+    chat.messages = [
     chat.messages = [
```

Comment:
I noticed something weird happened here. It looks like this one line is just a repeat. I think once this is fixed we should be good to go

Reply:
It looks like the duplicate line is still there. You just need to delete line 150 and that will take care of the issue.